readme.md

```diff
@@ -0,0 +1,10 @@
+# Savannah Safari Attractions - UoL Team Web Dev 41
+
+## Setup
+
+To lunch a server locally you can run `python -m http.server` and visit http://0.0.0.0:8000/
+Alternatively you can use [VSCode + Live server extension](https://github.com/ritwickdey/vscode-live-server) or [Brackets](http://brackets.io/)
```

Comment:
I added a link to this plugin if you decide to try VSCode, but miss the live preview from Brackets.

Reply:
Will give it a go next week...